src/net.cpp

```diff
@@ -1512,16 +1512,10 @@ void CConnman::SocketHandler()
             }
         }
 
-        //
-        // Send
-        //
-        if (sendSet)
-        {
-            LOCK(pnode->cs_vSend);
-            size_t nBytes = SocketSendData(pnode);
-            if (nBytes) {
-                RecordBytesSent(nBytes);
-            }
+        if (sendSet) {
+            // Send data
```

Comment:
in the first commit: I think the code is already self-documenting. No need for this

Reply:
I only kept this because there was a comment before but you're right - it's useless. Removing.